integr8ly/tutorial-web-app
# INTLY-418 Use new loading screen

## Motivation
The current loading screen being used does not display enough
information about the services being provisioned, the percentage
usually stalls at a percentage as one service takes much longer to
provision than others. This leads the end-users of the web app to
think that something has gone wrong.

## What
The new loading screen has a per-service status view which is a
better indicator of what's going on in the background.

## Verification Steps
* Login to the web app as a user with no Shared namespace
* Ensure the loading screen works as expected
